examples/scripts/configure-solr.sh

```diff
@@ -7,29 +7,27 @@
 # `solr` in the `installed_extras`, and is currently set up for the D7 versions
 # of Apache Solr Search or Search API Solr.
 
-SOLR_SETUP_COMPLETE_FILE=/etc/drupal_vm_solr_config_complete
+NAME="drupalvm"
```

Comment:
I like this change—one thing though, let's set the default to `d8`, just because that's what the D8 version of the Search API Solr Defaults module sets up in it's default index.

This would make the setup pretty much plug-and-play, as you'd immediately have a populated and working search index and search page after enabling the defaults module, as shown here: http://www.jeffgeerling.com/blog/2017/setting-faceted-apache-solr-search-drupal-8